Aliaksandr_Radyna/05. Ansible.start/README.md

```diff
@@ -0,0 +1,145 @@
+# 05. Ansible start
+## Homework Assignment 1: Setting Up Ansible
+### 1. Install Ansible on your local machine or a virtual environment.
+```bash
+adminn@ubuntuansible:~$ sudo apt install ansible
+
+adminn@ubuntuansible:~$ ansible --version
+ansible 2.9.6
```

Comment:
please use pip installation for ansible, old version of linux system has old python and ansible